app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -75,6 +79,12 @@
       if (changes.bindings) {
         row.deleteableBindings = _.reject(row.bindings, 'metadata.deletionTimestamp');
       }
+      if(row.isMobileService(row.apiObject) && $rootScope.AEROGEAR_MOBILE_ENABLED){
+        var integrations = _.get(getServiceClass(), "spec.externalMetadata.integrations", false);
```

Comment:
You don't need to give a default here since undefined is falsey. If you do, I'd rather it be `''` since we're expecting a string, but better to leave it off.

```js
var serviceClass = getServiceClass();
var integrations = _.get(serviceClass, "spec.externalMetadata.integrations");
```